packages/deployment/scripts/install-deps.sh

```diff
@@ -0,0 +1,38 @@
+#! /bin/bash
+set -e
+
+# Install Terraform.
+TERRAFORM_VERSION=0.11.14
+
+UNAME_S=$(uname -s | tr A-Z a-z)
+
+case $UNAME_S in
+*) TERRAFORM_OS=$UNAME_S ;;
+esac
+
+UNAME_M=$(uname -m)
+case $UNAME_M in
+x86_64) TERRAFORM_ARCH=amd64 ;;
+*) TERRAFORM_ARCH=$UNAME_M ;;
+esac
+
+TERRAFORM_RELEASE=terraform_${TERRAFORM_VERSION}_${TERRAFORM_OS}_${TERRAFORM_ARCH}
+curl https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/${TERRAFORM_RELEASE}.zip > terraform.zip && \
```

Comment:
I'm not sure of the context in which this script is run, but double-check that it's okay to create potentially large files in whatever the current directory is. `unzip` can't extract from stdin? Lame!

Reply:
Changed to use a temporary file with mktemp.